build/sail.tcl

```diff
@@ -16,3 +16,102 @@ respond "*" ".jbsa/strt\r"
 respond ":" "56/1000\r"
 respond "\n" "\033y sail; fail bin\r"
 respond "*" ":kill\r"
+
+# Make links needed for SUDS.
+respond "*" ":link wl; switch 999999, draw; switch >\r"
+respond "*" ":link wl; first 999999, draw; first >\r"
+respond "*" ":link wl; sigsub 999999, draw; sigsub >\r"
+respond "*" ":link draw; board0 999999, wl; board0 >\r"
+respond "*" ":link draw; board1 999999, wl; board1 >\r"
+respond "*" ":link draw; board2 999999, wl; board2 >\r"
+respond "*" ":link draw; decloc 999999, wl; decloc >\r"
+respond "*" ":link draw; lg627 999999, wl; lg627 >\r"
+respond "*" ":link draw; lg684 999999, wl; lg684 >\r"
+respond "*" ":link draw; mpg21 999999, wl; mpg21 >\r"
+respond "*" ":link draw; mpg216 999999, wl; mpg216 >\r"
+respond "*" ":link draw; ncp13 999999, wl; ncp13 >\r"
+respond "*" ":link draw; ug61c 999999, wl; ug61c >\r"
+respond "*" ":link sys1; ts d, datdrw; d bin\r"
+respond "*" ":link sys1; ts wl, datdrw; wl bin\r"
+respond "*" ":link sys1; ts pc, datdrw; pc bin\r"
+
+# The drawing program needs WL; BOARDS REL
+respond "*" ":cwd wl\r"
+respond "*" ":fail boards\r"
+respond ";^C" ";\003"
+
+# The PC program needs WL; PCBOARDS REL
+respond "*" "PCBOARDS_BOARDS\r"
+respond ";^C" "PC\r"
+respond "SELECTED" ";\003"
+respond "*" "\032:kill\r"
+
+# SUDS drawing program
+respond "*" ":cwd draw\r"
+respond "*" ":fail d\r"
+respond ";^C" ";\003"
+respond "*" "\032:kill\r"
+respond "*" ":stink d\r"
+respond "STINK." "\177"
+respond "??" "\033\0331L decsys; decbot bin\r"
```

Comment:
This stanza to dump out a file with the DECUUO bootstrap gets repeated several times - worth adding a helper function for it?

Reply:
Maybe.  And in general there are many other places in the build scripts that could benefit from calling helper functions.  I'm not exactly in love with TCL, so I haven't been eager to work on the scripts.  On the other hand, no other great solution has presented itself so maybe we're stuck with expect.